java/src/jmri/jmrix/ecos/utilities/AddRosterEntryToEcos.java

```diff
@@ -29,12 +31,15 @@
   */
 public class AddRosterEntryToEcos extends AbstractAction {
 
+    private EcosLocoAddressManager objEcosLocoManager;
+
     /**
      * @param s Name of this action, e.g. in menus
      */
     public AddRosterEntryToEcos(String s, EcosSystemConnectionMemo memo) {
         super(s);
         adaptermemo = memo;
+        objEcosLocoManager = adaptermemo.getLocoAddressManager();
```

Comment:
Cool! Meet to see FindBugs integration working.  

@kjlisby I'll create a fix for this shortly and give you instructions for getting merged. Thanks!